src/test/java/com/fasterxml/jackson/dataformat/yaml/SimpleParseTest.java

```diff
@@ -325,4 +325,38 @@ public void testObjectWithNumbers() throws Exception
 
         jp.close();
     }
+
+    public void testNulls() throws Exception
+    {
+        String YAML = "nulls: [!!null \"null\" ]";
+        JsonParser jp = YAML_F.createParser(YAML);
+
+        assertToken(JsonToken.START_OBJECT, jp.nextToken());
+        assertToken(JsonToken.FIELD_NAME, jp.nextToken());
+        assertEquals("nulls", jp.getCurrentName());
+        assertToken(JsonToken.START_ARRAY, jp.nextToken());
+        assertToken(JsonToken.VALUE_NULL, jp.nextToken());
+        assertToken(JsonToken.END_ARRAY, jp.nextToken());
+        assertToken(JsonToken.END_OBJECT, jp.nextToken());
+        assertNull(jp.nextToken());
+
+        jp.close();
+    }
+
+    public void testTildeNulls() throws Exception
+    {
+        String YAML = "nulls: [~ ]";
```

Comment:
I must have been reading updates in somewhat arbitrary order. I do see ref to http://yaml.org/spec/1.1/#id861700 thanks!